lib/vdsm/network/netinfo/routes.py

```diff
@@ -83,7 +83,7 @@ def get_gateway(
     gateways = [
         r
         for r in routes
-        if r['destination'] == 'none'
+        if r['destination'] in ('none', '0.0.0.0/0', '::/0')
```

Comment:
nit: Using sets specifically (i.e., with curly braces) seems to be more performant. Not sure how often this code is executed, so feel free to ignore the comment, but for the sake of experimentation, I did try:

```python
from timeit import timeit

dest = 'some'
print(timeit(lambda: dest in ['none', '0.0.0.0/0', '::/0'], number=10000000))
print(timeit(lambda: dest in {'none', '0.0.0.0/0', '::/0'}, number=10000000))
print(timeit(lambda: dest in ('none', '0.0.0.0/0', '::/0'), number=10000000))
print(timeit(lambda: dest == 'none' or dest == '0.0.0.0/0' or dest == '::/0', number=10000000))
```
Result:
```
8.609254561990383
5.435367603000486
7.046174183997209
9.278653385001235
```

Reply:
Afaik only on network sync and update of capabilities.
I'm happy to change it, but seems like we use ( ) everywhere in the code, and nowhere { }

Reply:
I'm more for consistency than this optimization that most likely doesn't have any impact overall.